components/public-api/gitpod/v1/workspace.proto

```diff
@@ -38,6 +43,28 @@ message WatchWorkspaceStatusResponse {
   WorkspaceStatus status = 2;
 }
 
+message ListWorkspacesRequest {
+  // pagination contains the pagination options for listing workspaces
+  PaginationRequest pagination = 1;
+
+  // organization_id is the ID of the organization that contains the workspaces
+  string organization_id = 2;
+
+  // pinned indicates whether to list only pinned workspaces
+  bool pinned = 3;
+
+  // search_term is a search term to filter workspaces by name
+  string search_term = 4;
```

Comment:
What is this actually searching? Is this the ID, description, repo details or something different? I think the comment should make it clear.

Reply:
`to filter workspaces by name` by name

Reply:
@mustard-mh in that case, can we please change to the following?
```suggestion
  // search_term is a search term to filter workspaces by description
  string search_term = 4;
```

Reply:
We named it `name` now

Reply:
Please keep it `searchString` and add a comment. The reason is that we want to stop creating a name from the context, but instead have no name and only when you pin a workspace you name it.
So we should really search by git status information and name (and id as long as we expose it like we do).

Reply:
@svenefftinge I don't understand. Do you mean to change comment that it can filter with `git status` , `workspaceId` and `name`?

Reply:
Oh, sorry. I missunderstood your comment that you named it `name` now as if you had changed the property name. So looks good now and we can update the docs once we change the implementation. Thank you!